README.md

```diff
@@ -104,25 +129,22 @@ When listening for other peers to connect, e.g. for executing a swap, a `peerd`
 To create an offer and spawn a listening `peerd` accepting incoming connections, run the following command:
 
 ```
-swap-cli make tb1q935eq5fl2a3ajpqp0e3d7z36g7vctcgv05f5lf\
-    54EYTy2HYFcAXwAbFQ3HmAis8JLNmxRdTC9DwQL7sGJd4CAUYimPxuQHYkMNg1EELNP85YqFwqraLd4ovz6UeeekFLoCKiu\
-    Testnet ECDSA Monero\
-    "0.00001350 BTC" "0.001 XMR"\
-    Alice 4 5 "1 satoshi/vByte"\
-    1.2.3.4\
-    0.0.0.0\
-    9735
+swap-cli make --arb-addr tb1q935eq5fl2a3ajpqp0e3d7z36g7vctcgv05f5lf\
```

Comment:
I'd prefer an example with all flags either here, or in local-swap, even if they are just passing default values.

Reply:
Done in 4a9baf3d586117f0632e22a909491a09ac6a1da0